docs/usage/13-lifecycle-hooks.md

```diff
@@ -0,0 +1,90 @@
+# Life Cycle Hooks
+
+Life cycle hooks allows a user to execute a function at a certain point, as indicated by the hook's name, during the
+request-response cycle.
+
+## Before Request
+
+The `before_request` hook runs immediately before calling the route handler function. It accepts either a sync or async
+function that receives the `starlite.connection.Request` instance as its sole parameter. While the handler function
+doeds not need to return a value, if it does return a value other than `None`, then the route handler will not be called
```

Comment:
Just someone monitoring this project from a distance but seem to have a minor typo here doeds -> does

Reply:
fixed